Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:bionic
```

Comment:
(Adding a comment so this PR doesn't get auto-closed by stalebot)

Just noticed this dockerfile - could we include this in a separate PR since it's not directly related to the route table work?